tests/e2e/utils.go

```diff
@@ -126,3 +126,51 @@ func matchResponseSubString(url, match string, retry, sleep int) bool {
 	fmt.Printf("Could not get the match string \"%s\" in %d seconds\n", match, i)
 	return false
 }
+
+// this function returns build name of a delopyed component by passing
+// component name as a argument
+func getBuildNameUsingOc(compName string) string {
```

Comment:
is`UsingOc` in every function really necessary?

Reply:
Yah, not really. 

Actually the ```oc``` command is being used to fetch the data. So i made the function name in that way. Also it looks more specific that we are executing using ```oc``` not ```odo```. IMO the function name can be more logical by changing the name like ```ocGetBuildName``` WDYT ?

Reply:
You don't need to mention oc at all. It is clear that it is using oc and not odo. There are no builds in odo. The longer name function has the harder it is to read the code. 

If you have long name like:
```
getBuildNameUsingOc("component")
```

it takes more efford to read it compared to something like this:
```
getBuildName("component")
```


The fact that it is using oc can be mentioned in function description (comment) not in a name

Reply:
hmm, make sense to me 

Thanks for the feedback

Reply:
Done